src/Flexbox.jsx

```diff
@@ -1,39 +1,38 @@
-import React, { PropTypes } from 'react';
+import PropTypes from 'prop-types';
+import React from 'react';
 import styled from 'styled-components';
 
-const { string, oneOf, node, number, bool, object } = PropTypes;
-
 const Flexbox = styled(({
-  alignContent,
-  alignItems,
-  alignSelf,
+  alignContent, // eslint-disable-line no-unused-vars
+  alignItems, // eslint-disable-line no-unused-vars
+  alignSelf, // eslint-disable-line no-unused-vars
   children,
-  display,
+  display, // eslint-disable-line no-unused-vars
   element,
-  flex,
-  flexBasis,
-  flexDirection,
-  flexGrow,
-  flexShrink,
-  flexWrap,
-  height,
-  justifyContent,
-  margin,
-  marginBottom,
-  marginLeft,
-  marginRight,
-  marginTop,
-  maxHeight,
-  maxWidth,
-  minHeight,
-  minWidth,
-  order,
-  padding,
-  paddingBottom,
-  paddingLeft,
-  paddingRight,
-  paddingTop,
-  width,
```

Comment:
I've noticed this error only rises in the editor, but not when the linting is done with the cli or in CI. That's why I didn't disabled those lines. It's weird. I still think we need to find a way to refactor this so we don't need to disable rules (of course without passing unknown props to the target component). Any ideas?